app/assets/locales/messages_ccodk_default.txt

```diff
@@ -304,6 +304,8 @@ demo.mode.feature.unavailable=Sorry, this app uses a feature that is not availab
 
 select.search.label=Search:
 select.search.status=${0} of ${1} results for your search: "${2}"
+select.callout.search.status=${0} of ${1} results for callout result

```

Comment:
This feature is isolated such that you have to enable a feature flag on HQ to trigger this behavior (that feature flag doesn't even exist yet). Nobody will see it unless I send them a custom ccz. Among other reasons, I really want to get it into 2.28 so that we can run UAT without sending out custom builds.